src/main/java/homework_6/map_problems_generator/problem_impl/GettingProblem.java

```diff
@@ -0,0 +1,15 @@
+package homework_6.map_problems_generator.problem_impl;
+
+import java.util.Random;
+
+public class GettingProblem <T> extends NoProblem <T> {
+
+    public GettingProblem(T value) {
+        super(value);
+    }
+
+    @Override
+    public int hashCode() {
+        return new Random().nextInt();
```

Comment:
I reread the task description, and I agree it's unclear there. But the task I had in mind is when you put mutable key, change it's field, and try to get value -> you can't, because hash is now different, even though hash was written according contracts.

Reply:
add this case(new class add)